src/jobs/controllers/taskController.ts

```diff
@@ -63,7 +63,7 @@ export class TaskController {
   };
   public getResources: GetResourcesHandler = async (req, res, next) => {
     try {
-      const tasksRes = await this.manager.getAllTasks(req.params);
+      const tasksRes = await this.manager.getAllTasks(req.params, !!req.query.shouldExcludeParameters);
```

Comment:
you can define an query params derictly for the `GetResourcesHandler` that way:
![image](https://github.com/user-attachments/assets/309e0e87-e777-477f-bd4b-05d257c97e9c)
or even use type to declare it like: `GetTasksQueryParams`

and that way you can use it without the `!!` and it will be recognized by its type and not just as string:
![image](https://github.com/user-attachments/assets/b786af16-6381-4e5a-8f21-c2515e085492)

Reply:
Done